modules/ROOT/content-nav.adoc

```diff
@@ -159,13 +159,23 @@
 ** xref:backup-restore/copy-database.adoc[]
 
 * xref:authentication-authorization/index.adoc[]
-** xref:authentication-authorization/introduction.adoc[]
+* Manage users
+** xref:authentication-authorization/manage-users.adoc[]
```

Comment:
this looks odd to me, I feel like the `* Manage users` should not be there? cause it's in the same level as the intro and then everything else is under it

Reply:
This looks like a leftover from when I played with the navigation. It should be fixed now.